apps/voting/app/src/disputable-app-logic.js

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback, useMemo } from 'react'
```

Comment:
Ah, we should remove this from this PR :)

Reply:
haha sorry I commit this by mistake!